src/js/tech/html5.js

```diff
@@ -86,6 +87,12 @@ class Html5 extends Tech {
    * @method dispose
    */
   dispose() {
+    let tt = this.el().textTracks;
+
+    tt.removeEventListener('change', Fn.bind(this, this.handleTextTrackChange));
```

Comment:
I just realized we could probably just clean out our emulated text track list manually. However, if we remove the event listener asynchronously, it works fine too. Or, we can have the `removetrack` handler be smart and get added on `addtrack` and removed on `removetrack` if there are no longer any text tracks (which is what I tried to describe above).